docs/maps/maps-aggregations.asciidoc

```diff
@@ -68,9 +68,9 @@ To enable a blended layer that dynamically shows clusters or documents:
 
 [role="xpack"]
 [[maps-top-hits-aggregation]]
-=== Top hits per entity
+=== Display the most relevant documents per entity
 
-You can display the most relevant documents per entity, for example, the most recent GPS tracks per flight.
+Use *Top hits per entity* to display the most relevant documents per entity, for example, the most recent GPS tracks per flight route.
```

Comment:
Now title is repeat of this sentence. Is that a problem?

Reply:
What do you think @gchaps?

I don't think it's a problem, but it's obviously not normal practice.

Reply:
It does feel a little awkward to have"per entity" appear three times before the second sentence.  What about removing that sentence and going with this:

>Elasticsearch groups your data using a terms aggregation, and then accumulates the most relevant documents based on sort order for each entry using a top hits metric aggregation.
>
>To view the top hits per entity:

Reply:
Don't loss the example, "for example, the most recent GPS tracks per flight route." needs to be maintained

Reply:
>Elasticsearch groups your data using a terms aggregation, and then accumulates the most relevant documents based on sort order for each entry using a top hits metric aggregation.  This enables to view metrics such as the most recent GPS tracks per flight route.
>
>To view the top hits per entity:

Reply:
Can't use the word "metric" because its not an aggregation. Unlike an aggregation, top hits is actually showing documents, just a very small number of documents per entity, where entity is an airplane or truck.

Reply:
metrics > top hits?

Reply:
I think the text is fine because *Tops hits per entity* is a UI command.

Use *Top hits per entity* to display the most relevant documents per entity, for example, the most recent GPS tracks per flight route.